.templates/openhab/service.yml

```diff
@@ -1,18 +1,19 @@
 openhab:
-  image: "openhab/openhab:latest"
   container_name: openhab
+  image: "openhab/openhab:latest"
   restart: unless-stopped
   network_mode: host
-  #ports:
-  #- "4050:4050"
-  #- "4051:4051"
   volumes:
-    - "/etc/localtime:/etc/localtime:ro"
-    - "/etc/timezone:/etc/timezone:ro"
-    - "./volumes/openhab/addons:/openhab/addons"
-    - "./volumes/openhab/conf:/openhab/conf"
-    - "./volumes/openhab/userdata:/openhab/userdata"
+  - "/etc/localtime:/etc/localtime:ro"
+  - "/etc/timezone:/etc/timezone:ro"
+  - "./volumes/openhab/addons:/openhab/addons"
+  - "./volumes/openhab/conf:/openhab/conf"
+  - "./volumes/openhab/userdata:/openhab/userdata"
   environment:
     - OPENHAB_HTTP_PORT=4050
     - OPENHAB_HTTPS_PORT=4051
-    - EXTRA_JAVA_OPTS=-Duser.timezone=Etc/UTC"
+    - EXTRA_JAVA_OPTS: "-Duser.timezone=Europe/Berlin"
+  logging:
```

Comment:
Some users have reported issues if logging is set.

Reply:
Pushed update to PR to fixed egregious typo in EXTRA_JAVA_OPTS (**tested** this time). Commented-out logging options. Only added for consistency with experimental branch.